packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -1277,6 +1277,808 @@ curl -v -X DELETE \
+                "hints": {
+                    "allow": [
+                        "POST"
+                    ]
+                }
+            },
+            "self": {
+                "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/OKTA_ISSUE"
+            }
+        }
+    },
+    {
+        "notificationType": "OKTA_UPDATE",
+        "channels": [
+            "email"
+        ],
+        "status": "unsubscribed",
+        "_links": {
+            "subscribe": {
+                "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/OKTA_UPDATE/subscribe",
+                "hints": {
+                    "allow": [
+                        "POST"
+                    ]
+                }
+            },
+            "self": {
+                "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/OKTA_UPDATE"
+            }
+        }
+    },
+    {
+        "notificationType": "USER_DEPROVISION",
+        "channels": [
+            "email"
+        ],
+        "status": "unsubscribed",
+        "_links": {
+            "subscribe": {
+                "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/USER_DEPROVISION/subscribe",
+                "hints": {
+                    "allow": [
+                        "POST"
+                    ]
+                }
+            },
+            "self": {
+                "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/USER_DEPROVISION"
+            }
+        }
+    },
+    {
+        "notificationType": "REPORT_SUSPICIOUS_ACTIVITY",
+        "channels": [
+            "email"
+        ],
+        "status": "subscribed",
+        "_links": {
+            "unsubscribe": {
+                "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/REPORT_SUSPICIOUS_ACTIVITY/unsubscribe",
+                "hints": {
+                    "allow": [
+                        "POST"
+                    ]
+                }
+            },
+            "self": {
+                "href": "http://rain.okta1.com:1802/api/v1/roles/SUPER_ADMIN/subscriptions/REPORT_SUSPICIOUS_ACTIVITY"
+            }
+        }
+    }
+]
+```
+
+#### Get subscription of a Role with a specific type
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="get" url="/api/v1/roles/${roleType}/subscriptions/${notificationType}" />
+
+Get subscriptions of a Role with a specific type
+
+##### Request parameters
+
+
+| Parameter        | Description                                            | Param Type  | DataType                                    | Required  |
+| :--------------- | :----------------------------------------------------- | :---------- | :------------------------------------------ | :-------- |
+| roleType         | `type` of a Role                                       | URL         | [Role Type](#role-types)                    | TRUE      |
```

Comment:
some of the spacing in different tables seem off. I know that's not how it will be rendered but still it'll be nice to have rows/columns of the table use equal spacing

Reply:
It looked bad in git but much better in intelliJ with wider screen 😃